### ISSUE TITLE
multi output training

### ISSUE THREAD

**Issue by soans1994** [Saleh-I/keras_tutorial#2]
hello author,

i want to train a model to predict heatmap (mean square error loss) and binary segmentation (binary cross entropy loss).
i tried to train model using multi branch (2 branch duplicates for 2 output). but the the final output will favour for only one type of output. 
For example when i train using model.fit with equal loss weights, the output is good for heatmap, but binary mask output is wrong and gives pixels 1 for the regions similar to heatmaps.
And when i train using gradtape loop, the output is good for segmentation mask, but heatmaps are wrong and looks like masks.

how can i solve this, please give me your suggestion.

thank you
